components/ui/dev-mock-card.tsx

```diff
@@ -0,0 +1,84 @@
+import * as React from "react";
+import { cn } from "@/lib/utils";
+
+// Development-only mock card components for testing
+
+const DevMockCard = React.forwardRef<
```

Comment:
Is `DevMockCard` used anywhere? If not, let's remove.

Reply:
ah good catch, it was part of some development for the mock page but ended up just using the mock page directly and forgot to remove this

removed in [4dc9330](https://github.com/hawaiians/hawaiiansintech/pull/47/commits/4dc9330842ca05341cb26577d9d128bedbc40fb1)